scripts/test_translation_pipeline.py

```
@@ -0,0 +1,489 @@
+        manager.register_agent_capability("stats-agent", "en", "es")
+        
+        # Create a job
+        source_segments = ["Hello", "World"]
+        job_id = manager.create_translation_job(
+            document_id="stats-doc-1",
+            source_language="en",
+            target_language="es",
+            source_segments=source_segments
+        )
+        
+        manager.start_translation_job(job_id, ["stats-agent"])
```

Pylint (4.0.2)

[error] No value for argument 'target_languages' in method call

(E1120)